<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemAttributesClass</class>
 <widget class="QWidget" name="ItemAttributesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemAttributes</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="propertyArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="propertyContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>144</width>
        <height>268</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="propertyLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="valueArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="valueContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>143</width>
        <height>268</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="valueLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
